build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.2'
+        classpath 'com.android.tools.build:gradle:2.3.0-alpha1'
```

Comment:
2.3 added new lint features including the baseline and a whole [bunch of other stuff](http://tools.android.com/tech-docs/lint-in-studio-2-3).